src/main/java/mate/jdbc/service/impl/DriverServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package mate.jdbc.service.impl;
+
+import java.util.List;
+import mate.jdbc.dao.driver.DriverDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.DriverService;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private DriverDao driverDao;
+
+    public Driver create(Driver driver) {
+        return driverDao.create(driver);
+    }
+
+    public Driver get(Long id) {
+        return driverDao.get(id).orElse(null);
```

Comment:
I would recommend you not to return null as it is a bad practice and you have to check in your controller(main) that you didn't get null. It is better to throw a runtime exception

Reply:
You are not checking this in main